pkg/elb/cleanup.go

```diff
@@ -459,6 +490,30 @@ func elbExists(ctx context.Context, elbAPI DescribeLoadBalancersAPI, elbv2API De
 	return desc != nil, err
 }
 
+func describeApplicationLoadBalancer(ctx context.Context, elbv2API DescribeLoadBalancersAPIV2,
+	name string) (*elbv2types.LoadBalancer, error) {
+
+	response, err := elbv2API.DescribeLoadBalancers(ctx, &elasticloadbalancingv2.DescribeLoadBalancersInput{
+		Names: []string{name},
+	})
+	if err != nil {
+		if isELBNotFoundErr(err) {
```

Comment:
A not-found error returned from `elbv2API.DescribeLoadBalancers` will not pass `isELBNotFoundErr`. The error returned would be of type `elasticloadbalancingv2types.LoadBalancerNotFoundException` which is not handled by `isELBNotFoundErr`.

Reply:
thanks for catching this ... I added a new `isELBv2NotFoundErr` function that looks for the correct exception in the right package.